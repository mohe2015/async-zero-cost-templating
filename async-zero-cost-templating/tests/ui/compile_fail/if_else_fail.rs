use async_zero_cost_templating::html;

pub fn main() {
    html! {
        if condition {
            <button class="submit-link" type="submit"></button>
        }
    }
}
