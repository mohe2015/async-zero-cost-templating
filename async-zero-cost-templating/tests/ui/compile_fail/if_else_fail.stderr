error: expected identifier, found keyword `type`
 --> tests/ui/compile_fail/if_else_fail.rs:6:21
  |
6 |             <button type="submit"></button>
  |                     ^^^^
  |
note: while parsing attribute
 --> tests/ui/compile_fail/if_else_fail.rs:6:21
  |
6 |             <button type="submit"></button>
  |                     ^^^^^^^^^^^^^^^^^^^^^^^
note: while parsing element
 --> tests/ui/compile_fail/if_else_fail.rs:6:13
  |
6 |             <button type="submit"></button>
  |             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
note: while parsing child
 --> tests/ui/compile_fail/if_else_fail.rs:6:13
  |
6 |             <button type="submit"></button>
  |             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
note: while parsing children
 --> tests/ui/compile_fail/if_else_fail.rs:6:13
  |
6 |             <button type="submit"></button>
  |             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
note: while parsing then branch
 --> tests/ui/compile_fail/if_else_fail.rs:5:22
  |
5 |           if condition {
  |  ______________________^
6 | |             <button type="submit"></button>
7 | |         }
  | |_________^
note: while parsing if
 --> tests/ui/compile_fail/if_else_fail.rs:5:9
  |
5 | /         if condition {
6 | |             <button type="submit"></button>
7 | |         }
  | |_________^
note: while parsing top level child
 --> tests/ui/compile_fail/if_else_fail.rs:5:9
  |
5 | /         if condition {
6 | |             <button type="submit"></button>
7 | |         }
  | |_________^
note: while parsing top level children
 --> tests/ui/compile_fail/if_else_fail.rs:5:9
  |
5 | /         if condition {
6 | |             <button type="submit"></button>
7 | |         }
  | |_________^

error: expected one of: string literal, `if`, `for`, curly braces, `<`
 --> tests/ui/compile_fail/if_else_fail.rs:6:21
  |
6 |             <button type="submit"></button>
  |                     ^^^^
  |
note: while parsing child
 --> tests/ui/compile_fail/if_else_fail.rs:6:21
  |
6 |             <button type="submit"></button>
  |                     ^^^^^^^^^^^^^^^^^^^^^^^
note: while parsing children
 --> tests/ui/compile_fail/if_else_fail.rs:6:13
  |
6 |             <button type="submit"></button>
  |             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
note: while parsing then branch
 --> tests/ui/compile_fail/if_else_fail.rs:5:22
  |
5 |           if condition {
  |  ______________________^
6 | |             <button type="submit"></button>
7 | |         }
  | |_________^
note: while parsing if
 --> tests/ui/compile_fail/if_else_fail.rs:5:9
  |
5 | /         if condition {
6 | |             <button type="submit"></button>
7 | |         }
  | |_________^
note: while parsing top level child
 --> tests/ui/compile_fail/if_else_fail.rs:5:9
  |
5 | /         if condition {
6 | |             <button type="submit"></button>
7 | |         }
  | |_________^
note: while parsing top level children
 --> tests/ui/compile_fail/if_else_fail.rs:5:9
  |
5 | /         if condition {
6 | |             <button type="submit"></button>
7 | |         }
  | |_________^

error: expected one of: string literal, `if`, `for`, curly braces, `<`
 --> tests/ui/compile_fail/if_else_fail.rs:6:25
  |
6 |             <button type="submit"></button>
  |                         ^
  |
note: while parsing child
 --> tests/ui/compile_fail/if_else_fail.rs:6:25
  |
6 |             <button type="submit"></button>
  |                         ^^^^^^^^^^^^^^^^^^^
note: while parsing children
 --> tests/ui/compile_fail/if_else_fail.rs:6:13
  |
6 |             <button type="submit"></button>
  |             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
note: while parsing then branch
 --> tests/ui/compile_fail/if_else_fail.rs:5:22
  |
5 |           if condition {
  |  ______________________^
6 | |             <button type="submit"></button>
7 | |         }
  | |_________^
note: while parsing if
 --> tests/ui/compile_fail/if_else_fail.rs:5:9
  |
5 | /         if condition {
6 | |             <button type="submit"></button>
7 | |         }
  | |_________^
note: while parsing top level child
 --> tests/ui/compile_fail/if_else_fail.rs:5:9
  |
5 | /         if condition {
6 | |             <button type="submit"></button>
7 | |         }
  | |_________^
note: while parsing top level children
 --> tests/ui/compile_fail/if_else_fail.rs:5:9
  |
5 | /         if condition {
6 | |             <button type="submit"></button>
7 | |         }
  | |_________^

error: expected one of: string literal, `if`, `for`, curly braces, `<`
 --> tests/ui/compile_fail/if_else_fail.rs:6:34
  |
6 |             <button type="submit"></button>
  |                                  ^
  |
note: while parsing child
 --> tests/ui/compile_fail/if_else_fail.rs:6:34
  |
6 |             <button type="submit"></button>
  |                                  ^^^^^^^^^^
note: while parsing children
 --> tests/ui/compile_fail/if_else_fail.rs:6:13
  |
6 |             <button type="submit"></button>
  |             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
note: while parsing then branch
 --> tests/ui/compile_fail/if_else_fail.rs:5:22
  |
5 |           if condition {
  |  ______________________^
6 | |             <button type="submit"></button>
7 | |         }
  | |_________^
note: while parsing if
 --> tests/ui/compile_fail/if_else_fail.rs:5:9
  |
5 | /         if condition {
6 | |             <button type="submit"></button>
7 | |         }
  | |_________^
note: while parsing top level child
 --> tests/ui/compile_fail/if_else_fail.rs:5:9
  |
5 | /         if condition {
6 | |             <button type="submit"></button>
7 | |         }
  | |_________^
note: while parsing top level children
 --> tests/ui/compile_fail/if_else_fail.rs:5:9
  |
5 | /         if condition {
6 | |             <button type="submit"></button>
7 | |         }
  | |_________^

error: expected identifier
 --> tests/ui/compile_fail/if_else_fail.rs:6:36
  |
6 |             <button type="submit"></button>
  |                                    ^
  |
note: while parsing element
 --> tests/ui/compile_fail/if_else_fail.rs:6:35
  |
6 |             <button type="submit"></button>
  |                                   ^^^^^^^^^
note: while parsing child
 --> tests/ui/compile_fail/if_else_fail.rs:6:35
  |
6 |             <button type="submit"></button>
  |                                   ^^^^^^^^^
note: while parsing children
 --> tests/ui/compile_fail/if_else_fail.rs:6:13
  |
6 |             <button type="submit"></button>
  |             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
note: while parsing then branch
 --> tests/ui/compile_fail/if_else_fail.rs:5:22
  |
5 |           if condition {
  |  ______________________^
6 | |             <button type="submit"></button>
7 | |         }
  | |_________^
note: while parsing if
 --> tests/ui/compile_fail/if_else_fail.rs:5:9
  |
5 | /         if condition {
6 | |             <button type="submit"></button>
7 | |         }
  | |_________^
note: while parsing top level child
 --> tests/ui/compile_fail/if_else_fail.rs:5:9
  |
5 | /         if condition {
6 | |             <button type="submit"></button>
7 | |         }
  | |_________^
note: while parsing top level children
 --> tests/ui/compile_fail/if_else_fail.rs:5:9
  |
5 | /         if condition {
6 | |             <button type="submit"></button>
7 | |         }
  | |_________^

error: expected one of: string literal, `if`, `for`, curly braces, `<`
 --> tests/ui/compile_fail/if_else_fail.rs:6:36
  |
6 |             <button type="submit"></button>
  |                                    ^
  |
note: while parsing child
 --> tests/ui/compile_fail/if_else_fail.rs:6:36
  |
6 |             <button type="submit"></button>
  |                                    ^^^^^^^^
note: while parsing children
 --> tests/ui/compile_fail/if_else_fail.rs:6:13
  |
6 |             <button type="submit"></button>
  |             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
note: while parsing then branch
 --> tests/ui/compile_fail/if_else_fail.rs:5:22
  |
5 |           if condition {
  |  ______________________^
6 | |             <button type="submit"></button>
7 | |         }
  | |_________^
note: while parsing if
 --> tests/ui/compile_fail/if_else_fail.rs:5:9
  |
5 | /         if condition {
6 | |             <button type="submit"></button>
7 | |         }
  | |_________^
note: while parsing top level child
 --> tests/ui/compile_fail/if_else_fail.rs:5:9
  |
5 | /         if condition {
6 | |             <button type="submit"></button>
7 | |         }
  | |_________^
note: while parsing top level children
 --> tests/ui/compile_fail/if_else_fail.rs:5:9
  |
5 | /         if condition {
6 | |             <button type="submit"></button>
7 | |         }
  | |_________^

error: expected one of: string literal, `if`, `for`, curly braces, `<`
 --> tests/ui/compile_fail/if_else_fail.rs:6:37
  |
6 |             <button type="submit"></button>
  |                                     ^^^^^^
  |
note: while parsing child
 --> tests/ui/compile_fail/if_else_fail.rs:6:37
  |
6 |             <button type="submit"></button>
  |                                     ^^^^^^^
note: while parsing children
 --> tests/ui/compile_fail/if_else_fail.rs:6:13
  |
6 |             <button type="submit"></button>
  |             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
note: while parsing then branch
 --> tests/ui/compile_fail/if_else_fail.rs:5:22
  |
5 |           if condition {
  |  ______________________^
6 | |             <button type="submit"></button>
7 | |         }
  | |_________^
note: while parsing if
 --> tests/ui/compile_fail/if_else_fail.rs:5:9
  |
5 | /         if condition {
6 | |             <button type="submit"></button>
7 | |         }
  | |_________^
note: while parsing top level child
 --> tests/ui/compile_fail/if_else_fail.rs:5:9
  |
5 | /         if condition {
6 | |             <button type="submit"></button>
7 | |         }
  | |_________^
note: while parsing top level children
 --> tests/ui/compile_fail/if_else_fail.rs:5:9
  |
5 | /         if condition {
6 | |             <button type="submit"></button>
7 | |         }
  | |_________^

error: expected one of: string literal, `if`, `for`, curly braces, `<`
 --> tests/ui/compile_fail/if_else_fail.rs:6:43
  |
6 |             <button type="submit"></button>
  |                                           ^
  |
note: while parsing child
 --> tests/ui/compile_fail/if_else_fail.rs:6:43
  |
6 |             <button type="submit"></button>
  |                                           ^
note: while parsing children
 --> tests/ui/compile_fail/if_else_fail.rs:6:13
  |
6 |             <button type="submit"></button>
  |             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
note: while parsing then branch
 --> tests/ui/compile_fail/if_else_fail.rs:5:22
  |
5 |           if condition {
  |  ______________________^
6 | |             <button type="submit"></button>
7 | |         }
  | |_________^
note: while parsing if
 --> tests/ui/compile_fail/if_else_fail.rs:5:9
  |
5 | /         if condition {
6 | |             <button type="submit"></button>
7 | |         }
  | |_________^
note: while parsing top level child
 --> tests/ui/compile_fail/if_else_fail.rs:5:9
  |
5 | /         if condition {
6 | |             <button type="submit"></button>
7 | |         }
  | |_________^
note: while parsing top level children
 --> tests/ui/compile_fail/if_else_fail.rs:5:9
  |
5 | /         if condition {
6 | |             <button type="submit"></button>
7 | |         }
  | |_________^

error[E0425]: cannot find value `condition` in this scope
 --> tests/ui/compile_fail/if_else_fail.rs:5:12
  |
5 |         if condition {
  |            ^^^^^^^^^ not found in this scope
